examples/darwin-framework-tool/commands/provider/OTAProviderDelegate.mm

```diff
@@ -75,6 +75,11 @@ - (void)handleQueryImageForNodeID:(NSNumber * _Nonnull)nodeID
         _selectedCandidate.userConsentNeeded = _userConsentNeeded;
         NSLog(@"User Consent Needed: %@", _selectedCandidate.userConsentNeeded);
     }
+    if (_delayedActionTime && _selectedCandidate.status.integerValue == MTROtaSoftwareUpdateProviderOTAQueryStatusBusy) {
```

Comment:
This does not seem related to the commissioner node id?

Reply:
Yes, apologies. It is part of the changes necessary for cert. Would you like me to create another PR for this?

Reply:
Yes, please!